packages/react/src/composite/SegmentedControl/src/SegmentedControl.tsx

```
@@ -84,55 +66,62 @@ const SegmentedControl: React.FC<SegmentedControlProps> &
-              {buttonChildren}
-            </SegmentedControlButton>
-          );
-        }
-        // Return the item as is if it's not a button (skeleton case)
-        return item;
-      })}
-    </Box>
+    <SegmentedControlContext.Provider value={contextValue}>
+      <Box
+        aria-label="Segmented control"
+        role="group"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<fieldset>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)